### DISCUSSION TITLE
Not able to install PRT using pac

### DISCUSSION THREAD

**Discussion by chinuVymo1997** [microsoft/powerplatform-build-tools#616] [Q&A]
`ChinmayMahajan@VYMO-IT-LPT-563 ~ % pac tool prt
Microsoft PowerPlatform CLI
Version: 1.28.3+g4ba7dfd
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

Error: Not a valid command. Try running 'pac [command] help'.

Hints: 
  Parse failed on:            prt
  Current command:            tool
  Is this a known command?    No, was it misspelled?
  Is this a known argument?   No, was it misspelled?
  Was it quote wrapped?       No, be sure to wrap values that contain spaces.

`

I am getting this error while trying to install PRT using Power Apps CLI. Please help

**Comment by devkeydet** [comment]
PRT only works on Windows.  How did you install pac?  I can repro your error on linux.
![image](https://github.com/microsoft/powerplatform-build-tools/assets/4499296/d9548179-33e9-4a2e-9dfe-75879ecd9039)

**Comment by petrochuk** [comment]
Plugin registration tool is Windows only application because it is built with WPF UI.

**Comment by mfamphlett** [comment]
I also came across this issue.

The tool should mention this when running `pac tool list`. I.e. something like 'platform unsupported' instead of stating the tool will be installed on first launch:

![image](https://github.com/user-attachments/assets/d9945f42-1c8b-4a6c-b6b2-67df163fa832)

It should probably also have a more helpful error message when running `pact tool xxx` on an unsupported platform.
